packages/compartment-mapper/src/bundle.js

```diff
@@ -206,60 +223,24 @@ export const makeBundle = async (read, moduleLocation, options) => {
     module.index = index;
     modulesByKey[module.key] = module;
   }
+  const parsersInUse = new Set();
   for (const module of modules) {
     module.indexedImports = Object.fromEntries(
       Object.entries(module.resolvedImports).map(([importSpecifier, key]) => [
         importSpecifier,
         modulesByKey[key].index,
       ]),
     );
+    parsersInUse.add(module.parser);
+    // TODO: ask for a better name:
+    module.yummyChunks = processModule(module);
```

Comment:
Any ideas what a collective noun of these thingamabobs should be?
Most names I introduced here are horrible and need thinking.

Reply:
`chonkyChunks`

Reply:
I’m going to go out on a limb with `module.processed`, and only better than that if we rename `processModule` around the name of the thing it returns instead of what it accepts. Maybe `module.generator` and `makeModuleGenerator(module)`.